backend/src/API/siteIntegrationAPI.ts

```diff
@@ -5,6 +5,12 @@ import { PermissionError, BadRequestError } from '../utils/errors';
 
 require('dotenv').config();
 
+/**
+ * Triggers a dispatch event to pull changes from the IDOL backend.
```

Comment:
I'd be a little more specific about what this does. 

e.g. Triggers a dispatch event for the pull-from-idol GitHub action. This job (blah blah blah brief description of what the job does).